gameObjects/planet.py

```diff
@@ -11,6 +11,7 @@ def __init__(self, name: str):
         self.__y: float = 0.0
         self.__starbaseLevel: int = 0
         self.__spaceStructureSlots: int = 0
+        self.__shipyardLevel: int = 0
```

Comment:
Shouldn't this be `str` here rather than `int`?  Shipyard level is a text string not a number.  Ditto at lines 73 and 77 below?